Trang-Doan.md

```diff
@@ -0,0 +1,2 @@
+Hey, this is Trang Doan.
+My middle name is Creativity.
```

Comment:
huh. that's cool. But where's the fancy markdown though?

Reply:
fixing